src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin circle($circle-size) {
+  height: $circle-size;
+  width: $circle-size;
+  border-radius: 50%;
+}
+
+@mixin center($center-size) {
+  height: 5%;
+  width: 5%;
+  border-radius: 50%;
+}
```

Comment:
Same, you're using this mixin only once

Reply:
same, maby in the future i'll reuse this mixin